fusor-ember-cli/app/templates/review/summary.hbs

```diff
@@ -14,9 +14,18 @@
 
     {{#if isOpenStack}}
         {{#accordion-item name="Red Hat Enterprise Linux - OpenStack Platform" isOpen=isOpenStackOpen}}
-            {{review-link label='Director URL' value=undercloudUrl isRequired=true isExternalURL=true}}
-            {{review-link label='Director username' value=undercloudUsername isRequired=true}}
-            {{review-link label='Director password' value=undercloudPassword isRequired=true}}
+            {{review-link label='Director URL'
+                          value=undercloudUrl
+                          isExternalURL=true}}
+            {{review-link label='Director username'
+                          value=undercloudUsername}}
+            {{review-link label='Director password'
+                          value=undercloudPassword}}
+            {{review-link label='Overcloud URL'
```

Comment:
I've only ever heard it referred to as the Overcloud so I think 'Overcloud URL' is appropriate. I think we would want display the url, username, and password for the Overcloud just as we do for the director/undercloud.